apps/meteor/server/lib/rooms/roomTypes/direct.ts

```diff
@@ -94,16 +94,20 @@ roomCoordinator.add(DirectMessageRoomType, {
 	async getNotificationDetails(room, sender, notificationMessage, userId) {
 		const useRealName = settings.get<boolean>('UI_Use_Real_Name');
 
+		const roomName = room.name || (await this.roomName(room, userId));
+
 		if (this.isGroupChat(room)) {
 			return {
-				title: await this.roomName(room, userId),
+				title: roomName,
```

Comment:
The `roomName` function currently returns the `fname` (and not the `name`) property when `useRealName` is `true`. Shouldn't we keep this behavior?
If so, then mabe we can remove the changes from this line :thinking:

Reply:
Done!